security/advancedtls/advancedtls.go

```diff
@@ -54,17 +54,34 @@ type VerificationFuncParams struct {
 	Leaf *x509.Certificate
 }
 
-// VerificationResults contains the information about results of
-// CustomVerificationFunc.
-// VerificationResults is an empty struct for now. It may be extended in the
+// DEPRECATED: Renamed to `HandshakeVerificationInfo`
```

Comment:
See other PR re: how Go writes "deprecated" docstrings.  This ties in with tooling in godoc and static checkers, so isn't just opinion.

Reply:
Fixed